packages/prisma/schema.prisma

```diff
@@ -499,6 +500,7 @@ model Team {
   bookingLimits                Json?
   includeManagedEventsInLimits Boolean                 @default(false)
   internalNotePresets          InternalNotePreset[]
+  credits                      CreditBalance?
```

Comment:
Let's use `creditBalance` here as well

could be confusing because `credits` (Int type) exists in CreditExpenseLog